desktop/core/src/desktop/urls.py

```diff
@@ -246,17 +246,17 @@
 for app in appmanager.DESKTOP_MODULES:
   if app.urls:
     if app.is_url_namespaced:
-      namespace = {'namespace': app.name, 'app_name': app.name}
+      namespace = app.name
     else:
-      namespace = {}
+      namespace = None
     if namespace or app in appmanager.DESKTOP_APPS:
-      app_urls_patterns.append(url('^' + re.escape(app.name) + '/', include(app.urls, **namespace)))
+      app_urls_patterns.append(url('^' + re.escape(app.name) + '/', include(app.urls, namespace=namespace)))
```

Comment:
For the record is seems that either we have an `app_name` in the urls.py of each app or specify it the list of urls https://docs.djangoproject.com/en/3.1/topics/http/urls/#namespaces-and-include

If nothing breaks now this is good, and we will revisit it anyway soon as it looks like 'url' is going away in next releases (replaced by 'path')

Reply:
Just checked, Deprecation warning is still there (somehow missed), tried some method either some tests are failing or the warning remains.

Reply:
https://app.circleci.com/pipelines/github/cloudera/hue/3847/workflows/546ad3ce-09b7-4cb6-a01d-509ead7bd09b/jobs/5192 not understanding why these tests are not passing after (include((app.urls, app.name), namespace=namespace))

Reply:
Yes, cf comment just above, `app_name` was removed and might be needed. Would recommend to try in one of the failing app